.vscode/settings.json

```
@@ -0,0 +1,5 @@
+{
+  "prettier.jsxSingleQuote": true,
+"prettier.singleQuote": true,
```

just some spaces missing 😃 
```suggestion
  "prettier.singleQuote": true,
```